client/src/App.jsx

```diff
@@ -15,6 +15,8 @@ import LoginPage from "./pages/login";
 import CreatePage from "./pages/create-user";
```

Comment:
Since we changed the title of the Client table page from "My Clients" to "Clients", it also be changed here
<img width="1212" alt="Screenshot 2024-02-25 at 10 43 14 PM" src="https://github.com/uoftblueprint/tces/assets/58835213/04e950a8-8258-4547-ab69-cffb08a49c53">

Reply:
switched, thanks!